helix-term/src/commands.rs

```diff
@@ -1000,31 +1000,70 @@ fn goto_file_vsplit(cx: &mut Context) {
     goto_file_impl(cx, Action::VerticalSplit);
 }
 
+fn resolve_selected_path(
+    selection_text: String,
+    doc_path_parent: Option<&Path>,
+) -> Option<PathBuf> {
+    let trimmed = selection_text.trim();
```

Comment:
Should this trim be done in the outer function instead?

Reply:
This would duplicate it in two places.

If I tried to extract it from that function and have like a pipeline

1. trim
2. check if empty
3. resolve relative to document, if possible

that would also be duplicated in two places. Putting the three steps in a function again, would
lead to the same result I have now.

Maybe I could rename the function `trim_and_resolve` or so?

Reply:
Not sure what is the best approach here, but the current one seemed fine to me as well.